tests/phpunit/tests/rest-api/rest-taxonomies-controller.php

```diff
@@ -291,4 +291,26 @@ protected function check_taxonomies_for_type_response( $type, $response ) {
 		$this->assertEquals( count( $taxonomies ), count( $data ) );
 	}
 
+	public function test_get_for_taxonomy_reuses_same_instance() {
+		$this->assertSame(
+			get_taxonomy( 'category' )->get_rest_controller(),
+			get_taxonomy( 'category' )->get_rest_controller()
+		);
+	}
+
+	public function test_get_for_taxonomy_returns_terms_controller_if_custom_class_not_specified() {
+		register_taxonomy(
+			'test',
+			'post'
```

Comment:
```suggestion
			'post',
```

Fixing this syntax error, https://travis-ci.com/github/WordPress/wordpress-develop/jobs/348832082#L2611